src/functionalTest/java/uk/gov/hmcts/reform/wataskmanagementapi/controllers/PostTaskForSearchCompletionControllerCFTTest.java

```diff
@@ -102,7 +102,7 @@ public void should_return_200_with_appropriate_task_to_complete() {
                 .statusCode(HttpStatus.OK.value())
                 .contentType(APPLICATION_JSON_VALUE)
                 .body("tasks.size()", equalTo(1))
-                .body("tasks[0].permissions.values", hasItems("Read","Refer","Own","Manage","Cancel"))
+                .body("tasks[0].permissions.values", hasItems("Read","Refer","Own"))
```

Comment:
as per requirement, we will need to change the matcher to check this contains only the permissions relevant to the user